core/src/test/java/org/elasticsearch/ElasticsearchExceptionTests.java

```diff
@@ -67,7 +67,7 @@ public void testToXContent() throws IOException {
         // in the JSON. Since the stack can be large, it only checks the beginning of the JSON.
         assertExceptionAsJson(e, true, startsWith("{\"type\":\"exception\",\"reason\":\"foo\"," +
                 "\"caused_by\":{\"type\":\"illegal_state_exception\",\"reason\":\"bar\"," +
-                "\"stack_trace\":\"java.lang.IllegalStateException: bar"));
+                "\"stack_trace\":\"[bar]\\n\\tat org.elasticsearch."));
```

Comment:
The cause was wrapped within the (now removed) anonymous ElasticsearchException so it was rendered using ElasticsearchException.toString() which was wrong. I should not have modified this.